src/components/chart/component.js

```diff
@@ -40,13 +40,21 @@ class Chart extends PureComponent {
     config: PropTypes.shape({}).isRequired,
     className: PropTypes.string,
     handleMouseMove: PropTypes.func,
-    handleMouseLeave: PropTypes.func
+    handleMouseLeave: PropTypes.func,
+    onReady: PropTypes.func
   };
 
   static defaultProps = {
     className: '',
     handleMouseMove: null,
-    handleMouseLeave: null
+    handleMouseLeave: null,
+    onReady: null
+  }
+
+  componentDidMount() {
```

Comment:
It is an opinion, I would start using hooks and useEffect instead of componentDidMount

Reply:
I think you are right, but I will leave this refactor for later (when we get real data)